SPTDataLoader/SPTDataLoaderService.m

```diff
@@ -187,24 +184,34 @@ - (void)cancelAllLoads
 
 #pragma mark SPTDataLoaderRequestResponseHandlerDelegate
 
-- (nullable id<SPTDataLoaderCancellationToken>)requestResponseHandler:(id<SPTDataLoaderRequestResponseHandler>)requestResponseHandler
-                                                       performRequest:(SPTDataLoaderRequest *)request
+- (void)requestResponseHandler:(id<SPTDataLoaderRequestResponseHandler>)requestResponseHandler
```

Comment:
It makes more sense to combine `requestResponseHandler:performRequest:` and `performRequest:requestResponseHandler:`, this way it is confusing.